packages/smooth_app/lib/pages/prices/prices_proofs_page.dart

```diff
@@ -138,9 +139,26 @@ class _PricesProofsPageState extends State<PricesProofsPage>
                                     ),
                                   ),
                                 );
-                              }, // PriceProofPage
-                              child: _PriceProofImage(proof,
-                                  squareSize: squareSize),
+                              },
+                              // Used our new ProofGridItem widget
+                              child: ProofGridItem(
```

Comment:
Just put your change into the existing code, in that case in `_PriceProofImage`.